sokutei.h

```diff
@@ -0,0 +1,72 @@
+#ifndef SOKUTEI_BENCHMARK
+#define SOKUTEI_BENCHMARK
+
+#define MAX_COUNTERS 100
+#define COUNTER_LENGTH 32
+#define INT int
```

Comment:
Consider the macro name `INT`. While it's a perfectly legitimate name, there's an overly big chance of a name clash with some other library or stuff, what do you think? I think, it would be better to namespace macro names the same way as you did for functions.

Thus, for example, `INT` can be `SOKUTEI_INT`.

Another thing, I would take into consideration is the descriptiveness of these names. Without documentation it is quite hard to guess what `SOKUTEI_INT` exactly does. What do you think, what would be a better name for these macros (not just `INT`)?